chapters/annotations.tex

```diff
@@ -165,7 +165,9 @@ \subsubsection{Plot Curves}\doublelabel{plot-curves}
 \end{lstlisting}
 
 The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be
-component references refering to a scalar variable or \lstinline!time!.
+component references refering to a scalar variable or \lstinline!time!. If
+\lstinline!x! or \lstinline!y! does not designate a variable the behavior is
+unspecified.
```

Comment:
Something like:
```suggestion
\lstinline!x! or \lstinline!y! does not designate a scalar variable or \lstinline!time! the \lstinline!Curve! is
invalid.
```

Reply:
I think this makes sense.

Reply:
We tend to say _is an error_ rather than _is invalid_, so I think we need to also include the start of the sentence:
> It is an error if `x` or `y` does not designate a scalar variable.

Reply:
Discussion continued in a new suggestion.